### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by Vilina**
Added header component. optimized it for mobile view

### CODE REVIEW

File: components/Header.vue [hunk 1]
```diff
@@ -1,18 +1,103 @@
...
-        <li>
-          <NuxtLink to="/about">About</NuxtLink>
+        <li class="ml-auto mr-5 lg:w-full">
+          <div
+            class="mx-0.5 my-1 inline-block h-12 w-12 align-bottom lg:relative lg:w-full"
+          >
+            <form>
+              <input
+                class="lg:h-15 absolute right-12 z-10 h-12 w-0 border-0 bg-white pl-4 pr-12 outline-none transition-all focus:right-14 focus:w-[calc(100%-80px)] focus:rounded-lg focus:border focus:border-grey-100 focus:pr-4 lg:right-0 lg:w-full lg:rounded-lg lg:border lg:border-grey-40 lg:bg-grey-20 lg:focus:right-0 lg:focus:w-full lg:focus:border-grey-100 lg:focus:bg-white"
+                id="searchright"
```
Comment (RubenBarseghyan) [inline components/Header.vue]: suggestion: Maybe rename like headerSearch

File: components/Header.vue [hunk 2]
```diff
@@ -1,18 +1,103 @@
...
+                type="search"
+                name="q"
+                placeholder="Search"
+              />
+              <label
```
Comment (RubenBarseghyan) [inline components/Header.vue:66]: suggestion: replace before input for semantic HTML principles

File: components/Header.vue [hunk 3]
```diff
@@ -1,18 +1,103 @@
...
+        <li class="ml-auto mr-5 lg:w-full">
+          <div
+            class="mx-0.5 my-1 inline-block h-12 w-12 align-bottom lg:relative lg:w-full"
+          >
+            <form>
+              <input
+                class="lg:h-15 absolute right-12 z-10 h-12 w-0 border-0 bg-white pl-4 pr-12 outline-none transition-all focus:right-14 focus:w-[calc(100%-80px)] focus:rounded-lg focus:border focus:border-grey-100 focus:pr-4 lg:right-0 lg:w-full lg:rounded-lg lg:border lg:border-grey-40 lg:bg-grey-20 lg:focus:right-0 lg:focus:w-full lg:focus:border-grey-100 lg:focus:bg-white"
+                id="searchright"
+                type="search"
+                name="q"
```
Comment (RubenBarseghyan) [inline components/Header.vue]: question: name="q" what is "q"

**Comment by Vilina** [comment]
Close on link click

**Comment by Vilina** [comment]
Fix mobile buttons width in menu

**Comment by Vilina** [comment]
> Close on link click

314dfcd54dcd26bbdc92393d9f2d81f1b7fdf0b3

**Comment by Vilina** [comment]
> Fix mobile buttons width in menu

314dfcd54dcd26bbdc92393d9f2d81f1b7fdf0b3

File: components/Header.vue (hunk 1 above)
Comment (Vilina) [reply components/Header.vue]: 314dfcd54dcd26bbdc92393d9f2d81f1b7fdf0b3

File: components/Header.vue (hunk 3 above)
Comment (Vilina) [reply components/Header.vue]: 314dfcd54dcd26bbdc92393d9f2d81f1b7fdf0b3

File: components/Header.vue (hunk 2 above)
Comment (Vilina) [reply components/Header.vue:66]: 314dfcd54dcd26bbdc92393d9f2d81f1b7fdf0b3

**Review by RubenBarseghyan — APPROVED** [review]

**Review by GayaneKasparova — APPROVED** [review]

**Pull request merged**